cli/src/maintenance.rs

```diff
@@ -1356,6 +1375,37 @@ impl SolidoState {
             Some(self_slice_start_slot)
         }
     }
+
+    /// Return None if we observe we moved past `1 -
+    /// SolidoState::END_OF_EPOCH_THRESHOLD`%. Return Some(()) if the above
+    /// condition fails or `self.stake_unstake_any_time` is set to
+    /// `true`.
+    pub fn should_perform_maintenance_at_end_of_epoch(&self) -> Option<()> {
```

Comment:
The `_at_end_of_epoch` is a bit misleading then, because it can also return `Some(())` at the start of the epoch if the flag is set. How about something like `confirm_should_stake_unstake_in_current_slot`?

Reply:
Yeah, that's much better 👍